shell/initialize/index.js

```diff
@@ -174,29 +170,22 @@ async function createApp(ssrContext, config = {}) {
   // Make app available into store via this.app
   store.app = app;
 
-  const next = ssrContext ? ssrContext.next : (location) => app.router.push(location);
+  const next = (location) => app.router.push(location);
   // Resolve route
-  let route;
-
-  if (ssrContext) {
-    route = router.resolve(ssrContext.url).route;
-  } else {
-    const path = getLocation(router.options.base, router.options.mode);
 
-    route = router.resolve(path).route;
-  }
+  const path = getLocation(router.options.base, router.options.mode);
+  const route = router.resolve(path).route;
 
   // Set context to app.context
   await setContext(app, {
     store,
     route,
     next,
     error:           app.nuxt.error.bind(app),
-    payload:         ssrContext ? ssrContext.payload : undefined,
-    req:             ssrContext ? ssrContext.req : undefined,
-    res:             ssrContext ? ssrContext.res : undefined,
-    beforeRenderFns: ssrContext ? ssrContext.beforeRenderFns : undefined,
-    ssrContext
```

Comment:
i think this relates to `$ssrContext`, but couldn't prove as my instance died. once the pr that removes `fetch-server` merges could you take a look if other references can go to?

Reply:
Just removed all mentions to `ssrContext`. The remaining are on `fetch.server` which is about to be deleted on another PR.

Additionally, I removed another function `applySSRData` which wasn't doing anything at this point in time.

For another PR, we should search for all mentions to `ssr` on the codebase and check if they can be removed.